pkg/roles/rolemanager/middleware/authorization.go

```diff
@@ -64,6 +64,9 @@ func (ram RoleManagerAuthorizationMiddleware) AddRole(ctx context.Context, sessi
 	}); err != nil {
 		return roles.RoleProvision{}, err
 	}
+	if err := ram.authorizeMembers(ctx, session, optionalMembers); err != nil {
```

Comment:
Can we rename this to `validateMembers`

In that function we can return something message like 
`<Member ID> is not member of domain` for other entities
and 
`<Member ID> is not found` for domains